src/Message/Toast.elm

```diff
@@ -41,21 +42,42 @@ label =
 -}
 decoder : Decoder Toast
 decoder =
+    Decode.oneOf
+        [ v2decoder
+        , v1Decoder
+        ]
```

Comment:
To handle old versions, we create decoders for previous supported versions and use `Decode.oneOf` to just try them in order.